src/model/convLSTM/myConvLSTM_backup.py

```diff
@@ -137,7 +141,7 @@ def built_graph(self):
 #            self.X = tf.placeholder(tf.float32, shape=[None,cfg.NN_input['height'],cfg.NN_input['width']], name="X")
 #            self.X_reshaped = tf.reshape( self.X, shape=[-1, cfg.NN_input['height'], cfg.NN_input['width'], cfg.NN_input['channels']])     
 #            self.y = tf.placeholder(tf.int32, shape=[None,cfg.NN_Dense['n_outputs']], name="y")  
-            initializer = tf.contrib.layers.xavier_initializer()
+            initializer = tf2.initializers.GlorotUniform()
```

Comment:
The contrib layer is gone for v1 compat, but the GlorotUniform initializer should be idential to the xavier one